gradle/test.gradle

```diff
@@ -173,26 +173,36 @@ subprojects {
   }
 
   task repeatDistributedTest(type: RepeatTest) {
+    // Any dependencies added to the base test task must cascade to this task
+    dependsOn distributedTest
```

Comment:
Upon immediate hindsight, this is absolutely not the right way to do this.  Maybe examine the class, get a list of dependencies, and depend on those.  As it stands, our `repeat-new-tests.sh` will run the entire `*Test` instead of just repeating the changed tests within that scope.